packages/data/src/world/mq/bottom_of_the_well_mq.yml

```diff
@@ -59,7 +41,108 @@
+    "Bottom of the Well Basement": "true"
+  events:
+    BOTW_CENTER_WEST_TORCH: "true"
+"Bottom of the Well Main Room Center":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Wallmaster Main": "soul_wallmaster"
+    "Bottom of the Well Main Room Center Switch Corner": "has_hover_boots || trick(OOT_BOTW_MQ_BACKFLIP) || hookshot_anywhere || climb_anywhere"
+    "Bottom of the Well West Center Room": "event(BOTW_CENTER_WEST_TORCH)"
+    "Bottom of the Well East Center Room": "event(BOTW_CENTER_EAST_TORCH)"
+    "Bottom of the Well Basement Switch Ledge": "event(BOTW_MQ_GATE_FROM_SWITCH)"
+    "Bottom of the Well Basement": "true"
+  events:
+    BOTW_CENTER_EAST_TORCH: "can_hit_triggers_distance || can_hookshot || has_explosives || can_use_mask_blast"
+  locations:
+    "MQ Bottom of the Well Map Chest": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 1": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 2": "true"
+    "MQ Bottom of the Well Pot Lobby Cage 3": "true"
+    "MQ Bottom of the Well Lobby Cage Big Fairy": "can_play_sun"
+"Bottom of the Well Basement":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main": "true"
+    "Bottom of the Well Basement Switch Ledge": "climb_anywhere || hookshot_anywhere"
+    "Bottom of the Well Wallmaster Basement": "soul_wallmaster"
+  locations:
+    "MQ Bottom of the Well Lens Chest": "event(BOTW_MQ_REDEAD_CHEST) && (has_weapon || can_use_sticks || can_use_din || has_iron_boots || can_play_sun)" #Trying to get past these ReDeads/Wallmaster with actually nothing isn't fair.'
+    "MQ Bottom of the Well Basement Big Fairy": "can_play_sun"
+    "MQ Bottom of the Well GS Basement": "gs && can_damage_skull"
+    "MQ Bottom of the Well Heart Basement 1": "true"
+    "MQ Bottom of the Well Heart Basement 2": "true"
+    "MQ Bottom of the Well Heart Basement 3": "true"
+"Bottom of the Well West Center Room":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main Room Center": "true"
+  locations:
+    "MQ Bottom of the Well GS West Middle Room": "gs && can_damage"
+"Bottom of the Well East Center Room":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Main Room Center": "true"
+  locations:
+    "MQ Bottom of the Well East Middle Room Key": "true"
+    "MQ Bottom of the Well Pot Side Room 1": "true"
+    "MQ Bottom of the Well Pot Side Room 2": "true"
+    "MQ Bottom of the Well Pot Side Room 3": "true"
+    "MQ Bottom of the Well Wonder Item Side Room 1": "can_use_slingshot"
+    "MQ Bottom of the Well Wonder Item Side Room 2": "can_use_slingshot"
+    "MQ Bottom of the Well Wonder Item Side Room 3": "can_use_slingshot"
+    "MQ Bottom of the Well Wonder Item Side Room 4": "can_use_slingshot"
+"Bottom of the Well Basement Switch Ledge":
+  dungeon: BotW
+  exits:
+    "Bottom of the Well Basement": "true"
+    "Bottom of the Well Main Room Center": "climb_anywhere && hookshot_anywhere"
```

Comment:
does this actually work with the gate still there? Those gates have collision on both sides.

Reply:
added the gate event to mkae it work.